libsolidity/parsing/Parser.cpp

```diff
@@ -382,6 +382,34 @@ Declaration::Visibility Parser::parseVisibilitySpecifier()
 	return visibility;
 }
 
+ASTPointer<OverrideSpecifier> Parser::parseOverrideSpecifier()
+{
+	solAssert(m_scanner->currentToken() == Token::Override, "");
+	ASTNodeFactory nodeFactory(*this);
+
+	m_scanner->next();
+
+	std::vector<ASTPointer<UserDefinedTypeName>> overrides;
+
+	if (m_scanner->currentToken() == Token::LParen)
+	{
+		m_scanner->next();
+		while (m_scanner->currentToken() == Token::Identifier)
```

Comment:
What happens for let's say ``override(A, address)``?

Reply:
I think using `while (m_scanner->currentToken() != Token::RParen)` should fix that.

Reply:
That might still fail (resp. "fail to fail") for ``override(A,)``, right?

Reply:
Both should be test cases in any case.

Reply:
Also ``override()`` should be a test case - I'd disallow that as well, but we could also say it's fine.

Reply:
Oh of course, I did not mean that just modifying the while condition is the only required change.